client-side/src/components/statistics/graphBar.jsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import { useQuery } from '@apollo/client';
+import { BarChart } from '@mui/x-charts/BarChart';
+import { useAppSelector } from '../../redux/store.jsx';
+import Loader from '../../stories/loader/loader.jsx'
+import { getVisitedWebsitesByDate, formatDate } from './graphsUtils.js';
+import { GET_USERS } from './queries.js';
+
+export default function GraphBar({ startDate, endDate }) {
+  const users = useQuery(GET_USERS);
+  const user = useAppSelector((state) => state.user.currentUser);
+  startDate = formatDate(new Date(startDate.$d));
+  endDate = formatDate(new Date(endDate.$d));

```

Comment:
this rows are override the values that come from props, if you didn't need the props value, please remove it from there

Reply:
I need the accessories, I just used the same variable, is it better if I use a new variable?

Reply:
yes, it will be more clear if you will assign it to a new variables something like: formattedStartDate

Reply:
I change it.